src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
```

Comment:
Looks good. But maybe on another PR?

Could you create an issue as a remember?